ios/Runner/Info.plist

```diff
@@ -41,5 +41,11 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>App needs access to your location to start the hike</string>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>App needs access to your location to start the hike</string>
+	<key>CADisableMinimumFrameDurationOnPhone</key>
```

Comment:
Is there a drawback to doing this, such as more battery usage? Why isn't it the phone default?

Reply:
Yes, it can result in significant power consumption as higher refresh rates consume more battery power.
In Flutter 3.0.0 and newer versions, the CADisableMinimumFrameDurationOnPhone key is automatically added to the Info.plist file hence it is the phone default. (mentioned in changes made section of flutter 3.0 release notes )(https://docs.flutter.dev/development/tools/sdk/release-notes/release-notes-3.0.0)